src/babel-preset/node.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  presets: [
+    [
+      require.resolve('babel-preset-env'),
+      {
+        targets: {
+          node: 'current',
```

Comment:
Any thoughts on pulling this from package.json instead?  I mention that only as a single-source-of-truth thing, as I honestly can't think of a situation where `current` would be a problem.

Reply:
I request that we keep current because I take advantage of the fact that I can run Kibana server in node v8 and babel-preset-env doesn't transpile async/await and makes debugging so much easier